src/sql/parts/notebook/models/sparkMagicContexts.ts

```diff
@@ -76,13 +112,13 @@ export class SparkMagicContexts {
 		let defaultConnection: IConnectionProfile = SparkMagicContexts.DefaultContext.defaultConnection;
 		let activeConnections: IConnectionProfile[] = await connectionService.getActiveConnections();
 		// If no connections exist, only show 'n/a'
-        if (activeConnections && activeConnections.length > 0) {
-            // Remove all non-Spark connections
-            activeConnections = activeConnections.filter(conn => conn.providerName === notebookConstants.hadoopKnoxProviderName);
-        }
-        if (activeConnections.length === 0) {
-            return SparkMagicContexts.DefaultContext;
-        }
+		if (activeConnections && activeConnections.length > 0) {
```

Comment:
I ran formatter when copying in the JSON, seems to have altered these sections. Apologies for the confusion.